src/backend_common.cc

```diff
@@ -217,11 +217,15 @@ ReadInputTensor(
   RETURN_IF_ERROR(TRITONBACKEND_InputPropertiesForHostPolicy(
       input, host_policy_name, nullptr, nullptr, nullptr, nullptr,
       &input_byte_size, &input_buffer_count));
+  const char* request_id = "";
+  LOG_IF_ERROR(
+      TRITONBACKEND_RequestIdString(request, &request_id),
+      "unable to get request ID string");
```

Comment:
Can you wrap this into a function (i.e. `std::string GetRequestId(Request* req)`) in `backend_common.h`, and call the function in place of `std::string(request_id)`. The current handling is okay but we will want to avoid extra work in inference path and extract the request ID only when we need it (logging / returning error)

Reply:
Sure, done!